homeassistant/components/tellduslive.py

```diff
@@ -0,0 +1,139 @@
+"""
+homeassistant.components.tellduslive
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Tellduslive Component
+
+This component adds support for the Telldus Live service.
+Telldus Live is the online service used with Tellstick Net devices.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.tellduslive/
+
+Developer access to the Telldus Live service is neccessary
+API keys can be aquired from https://api.telldus.com/keys/index
+
+Tellstick Net devices can be auto discovered using the method described in:
+https://developer.telldus.com/doxygen/html/TellStickNet.html
+
+It might be possible to communicate with the Tellstick Net device
+directly, bypassing the Tellstick Live service.
+This however is poorly documented and yet not fully supported (?) according to
+http://developer.telldus.se/ticket/114 and
+https://developer.telldus.com/doxygen/html/TellStickNet.html
+
+API requests to certain methods, as described in
+https://api.telldus.com/explore/sensor/info
+are limited to one request every 10 minutes
+
+"""
+
+from datetime import timedelta
+import logging
+
+from tellcore.constants import (
```

Comment:
You cannot reference any of your requirements outside of the methods. As the file will be loaded before the requirements are installed.